src/Microsoft.AspNetCore.Server.IntegrationTesting/breakingchanges.netcore.json

```diff
@@ -0,0 +1,78 @@
+[
+    {
+      "TypeId": "public abstract class Microsoft.AspNetCore.Server.Testing.ApplicationDeployer : Microsoft.AspNetCore.Server.Testing.IApplicationDeployer",
```

Comment:
Went to confirm the timeline on https://www.nuget.org and realized we've never published either package. Breaking changes aren't relevant and API checks shouldn't be running here.

Revert changes in this project and, ideally, remove the baseline file.